BT-250 SIRIUS cron fix

intaabhay opened the pull request:
- Added a new cron for SIRIUS
- Edited the crontab accordingly
- Kept old crontab to be safe but commented out in crontab

Benoitdw reviewed (commented):
- Remove `| ALEX INFO |` - `| ALEX ERROR |` in log for only `| INFO |`  - `| ERROR |`
- Use comment to explain wha'ts your code do, not for commenting lines because it is versionned with git  (see second line in  crontab)
-  With same optic; don't name your file `new` or `old` just change the file - it's versionned anyway and easier to trace the modifications if they are all in the same file

intaabhay commented:
- Removed `ALEX` in logs
- Removed commented lines of code
- Removed extra files
- Changed behavior when PROJECTS variable was "null" (see attached image)
![Capture d’écran du 2023-01-03 13-52-56](https://user-images.githubusercontent.com/120393173/210360917-7f7c769b-06f7-41fc-8cab-d47a3fbf90cd.png)
